libs/examples/src/table/table-export-selection-example/table-export-selection-example.html

```diff
@@ -2,7 +2,7 @@
   [dataSource]="dataSource"
   dtSort
   dtTableSelection
-  showExportButton="true"
+  showExportButton="visible"
```

Comment:
Why is this change in the API necessary?

Reply:
See above for api change. This example was changed to show the missing table button.